IQKeyboardManagerSwift/IQTextView/IQTextView.swift

```diff
@@ -67,6 +67,7 @@ open class IQTextView : UITextView {
                     unwrappedPlaceholderLabel.lineBreakMode = .byWordWrapping
                     unwrappedPlaceholderLabel.numberOfLines = 0
                     unwrappedPlaceholderLabel.font = self.font
+                    unwrappedPlaceholderLabel.textAlignment = self.textAlignment
```

Comment:
I thing we should also override setTextAlignment method of the class and set the label text there also, because alignment may be change at runtime also.